plugins/mcreator-localization/lang/texts.properties

```diff
@@ -2484,6 +2484,7 @@ elementgui.living_entity.event_mob_kills_another=When this entity kills another
 elementgui.living_entity.event_mob_tick_update=On entity tick update
 elementgui.living_entity.event_player_collides_with=When player collides with this entity
 elementgui.living_entity.event_initial_spawn=On initial entity spawn
+elementgui.living_entity.celebration_sound=Celebration sound:
```

Comment:
Call this "Raid celebration sound"

Reply:
is it okay if I put on "raider celebration sound"? (as each raider plays his celebration sound)

Reply:
Yes

Reply:
Fixed